project/BinaryIncompatibilities.scala

```diff
@@ -6,6 +6,40 @@ object BinaryIncompatibilities {
   )
 
   val Tools = Seq(
+      ProblemFilters.exclude[MissingMethodProblem](
+        "org.scalajs.core.tools.linker.backend.emitter.ScalaJSClassEmitter.this"),
```

Comment:
Yes, that's the idea of binary compatible ;) Because for everything internal to Scala.js, we have control of all the binaries, and we recompile them all. But we must not break APIs that are available to a third party.